utils/rtty/files/rtty.init

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+
+BIN=/usr/sbin/rtty
+
+start_rtty() {
+    local cfg="$1"
+    local interface ifname id description host port ssl
+
+    uci_validate_section rtty rtty "${1}" \
+        'interface:uci("network", "@interface"):br-lan' \
```

Comment:
`br-lan` is linux system interface name, but the `interface` option expects OpenWrt logical interface name like `lan`, `wan`?

Reply:
Fixed.